### PR TITLE
Modified physics for more vanilla-like behavior

### PR DESCRIPTION
**Pull request by DevToaster**
I'm very sorry for the commit / pull request spam.  I was having trouble with Git not syncing correctly.  I fixed all of the previous issues and removed those mysterious merge tags and it now builds fine.

### CODE REVIEW

File: src/Entities/Entity.cpp [hunk 1]
```diff
@@ -943,6 +944,7 @@ void cEntity::HandlePhysics(std::chrono::milliseconds a_Dt, cChunk & a_Chunk)
 		{
 			// Normal gravity
 			fallspeed = m_Gravity * DtSec.count();
+			NextSpeed -= NextSpeed * m_AirDrag * DtSec.count();
```
Comment (madmaxoft) [inline src/Entities/Entity.cpp]: This calculation means that `m_AirDrag` is measured neither in "ratio per second" nor "ratio per tick".

Comment (DevToaster) [inline src/Entities/Entity.cpp]: I see, it's not exactly m_AirDrag / sec when compounded.  I'll modify it so it's m_AirDrag per interpolated tick.

**Comment by SphinxC0re** [comment]
any update on this one?

**Comment by madmaxoft** [comment]
Code-wise, I think this is good.

**Comment by SphinxC0re** [comment]
Well, then why not merge this?

**Comment by madmaxoft** [comment]
Has anyone else actually tried this branch, if it doesn't break anything? I'm a bit cautious about any physics code lately.

**Comment by DevToaster** [comment]
I've tested this myself.  It even fixes Blaze fireball projectiles slowing down and freezing midair in the overworld when shot upwards.

**Comment by madmaxoft** [comment]
And it doesn't mess with player movement?

**Comment by DevToaster** [comment]
Nope, not at all.
It seems that player movement is handled entirely client side anyways.  Mobs seem to have the same aerodynamics as players now.  The only thing I can think of is the "floaty" behavior of hobs when hit, but that's due to the raytrace/collision handling within the HandlePhysics function.

**Comment by madmaxoft** [comment]
Actually, if you happen to send a SetSpeed packet to the client, it starts moving in that direction (see #1300). But if you say it doesn't happen, then it's safe to merge.

**Comment by DevToaster** [comment]
Nothing involving SetSpeed has been changed from the master branch, just NextSpeed is subtracted by the air drag * NextSpeed.  It appears to work flawlessly.

**Pull request merged**